great_ape_safe/ape_api/ebtc.py

```diff
@@ -8,10 +9,19 @@
 C = Console()
 
 
+class CdpStatus(Enum):
+    # NOTE: there are more states, adding `2` to avoid magic numbers in code
+    CLOSED = 2
```

Comment:
good insight, in this particular case since its value its only issue to assert when in fact `closeCdp` is called proactively by a msig, i added only `2` to avoid inclusion of magic figures, so at least there is a referencing to its meaning. but you're right on the consideration of the other figures